core-ui/src/components/Extensibility/ds/widgets-form/NameRenderer.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import * as jp from 'jsonpath';
+
+import { ResourceForm } from 'shared/ResourceForm';
+import { K8sNameField } from 'shared/ResourceForm/fields';
+
+export function NameRenderer({ storeKeys, resource, setResource }) {
+  const propertyPath = `$.${storeKeys.toJS().join('.')}`;
+  const handleNameChange = name => {
+    jp.value(resource, '$.metadata.name', name);
+    jp.value(resource, "$.metadata.labels['app.kubernetes.io/name']", name);
+
+    setResource({ ...resource });
+  };
+  return (
+    <ResourceForm.Wrapper resource={resource} setResource={setResource}>
```

Comment:
same story as `KeyValueField` - maybe go with ui-schema handlers?

Reply:
If we change it then maybe together with KeyValueField